### DISCUSSION TITLE
Default user when using Bypass?

### DISCUSSION THREAD

**Discussion by ameeno** [authelia/authelia#7964] [Q&A]
Hello There.

I have setup authelia with my reverse proxy to bypass auth when I am on my lan at home, but require auth when I am remote.
Additionally, I use the following headers so my app can identify me:

```
Remote-Email: {http.reverse_proxy.header.Remote-Email}
Remote-Groups: {http.reverse_proxy.header.Remote-Groups}
Remote-Name: {http.reverse_proxy.header.Remote-Name}
Remote-User: {http.reverse_proxy.header.Remote-User}

```

Would it be possible to assign a default user to these headers when using bypass? otherwise I have to enable one_factor auth while at home so apps can identify me on lan!

**Comment by christaikobo** [comment]
The easier way is to config your reverse proxy to add fixed header when source IP is in local network.
